formula/phplint.rb

```diff
@@ -0,0 +1,66 @@
+class Phplint < Formula
+  desc "Validator and documentator for PHP 5 and 7 programs"
+  homepage "http://www.icosaedro.it/phplint/"
+  url "http://www.icosaedro.it/phplint/phplint-3.0_20160307.tar.gz"
+  version "3.0-20160307"
+  sha256 "7a361166d1a6de707e6728828a6002a6de69be886501853344601ab1da922e7b"
+
+  depends_on "php@7.1"
+
```

Comment:
Shouldn't this be prevented from being bottled?

Reply:
Yes it looks like this can be bottle unneeded. Good catch @SMillerDev

Reply:
True. I'll adjust.